packages/integration-angular/package.json

```diff
@@ -1,9 +1,10 @@
 {
-  "name": "@grafana/javascript-agent-integration-angular",
+  "name": "@grafana/agent-integration-angular",
   "version": "0.0.0",
   "description": "Grafana JavaScript Agent package that enables easier integration in projects built with Angular.",
   "license": "Apache-2.0",
   "author": "Grafana Labs",
+  "private": true,
```

Comment:
Is it `private` actually?

Reply:
would rather not release to npm until it has an implementation, so marked as private